src/fragments/lib/in-app-messaging/respond-interaction-events/respond-interaction-events.mdx

```diff
@@ -1,5 +1,10 @@
 Your code can respond with additional behavior to your users interacting with in-app messages by adding interaction event listeners.
 
+<Callout informational>
+  If you are using TypeScript for your development, you can import and use
+  the InAppMessageInteractionEvent enum.
```

Comment:
for what?
"you can import and use
  the InAppMessageInteractionEvent enum to filter the incoming messages with" <- Or some other action? Or are there multiple potential usages that we want to leave open for the reader?

Reply:
Thanks for bringing this up. This callout used to be near the bottom in the appropriate section (this enum is used for `notifyMessageInteraction`) and I don't really know how/why it got moved up. I'll move it back down.